components/molecule/notification/src/index.scss

```diff
@@ -35,9 +38,10 @@
     width: $sz-notification-icon;
 
     &Left {
-      @include media-breakpoint-down(xxs) {
+      @include media-breakpoint-down(xSmall) {
```

Comment:
we also decided to rename to the old breakpoint naming convention, so xxs should be ok. You might be rename the breakpoints here https://github.com/SUI-Components/sui-theme/blob/master/src/layout/_breakpoints.scss and include the _breakpoint file as it isn't being done yet

Reply:
i was told that the decision was not to change the naming convention. 
https://confluence.schibsted.io/pages/viewpage.action?spaceKey=ST&title=Breakpoints+-+Frontend

Reply:
https://github.com/SUI-Components/sui-theme/tree/master/src

Reply:
Great @jcger. 
your comment `we also decided to rename to the old breakpoint naming convention` was a bit confusing to me.